modules/library/src/main/scala/zio/elasticsearch/ElasticAggregation.scala

```diff
@@ -345,6 +345,26 @@ object ElasticAggregation {
   final def percentilesAggregation(name: String, field: String): PercentilesAggregation =
     Percentiles(name = name, field = field, percents = Chunk.empty, missing = None)
 
+  /**
+   * Constructs an instance of [[zio.elasticsearch.aggregation.SamplerAggregation]] using the specified parameters.
+   *
+   * @param name
+   *   the name of the aggregation
+   * @param agg
+   *   the first required sub-aggregation to be included in the sampler
+   * @param aggs
+   *   additional sub-aggregations to be included in the sampler
+   * @return
+   *   an instance of [[zio.elasticsearch.aggregation.SamplerAggregation]] that represents sampler aggregation to be
+   *   performed. This aggregation has a default `shard_size` of `100` documents per shard.
+   */
+  final def samplerAggregation(
+    name: String,
+    agg: SingleElasticAggregation,
+    aggs: Chunk[SingleElasticAggregation]*
```

Comment:
You should use `SingleElasticAggregation*` instead of `Chunk[SingleElasticAggregation]*`.